.betterer.results

```diff
@@ -4898,6 +4898,10 @@ exports[`better eslint`] = {
       [0, 0, 0, "Unexpected any. Specify a different type.", "2"],
       [0, 0, 0, "Unexpected any. Specify a different type.", "3"]
     ],
+    "public/app/features/inspector/troubleshooting.ts:5381": [
+      [0, 0, 0, "Unexpected any. Specify a different type.", "0"],
+      [0, 0, 0, "Unexpected any. Specify a different type.", "1"]
+    ],
```

Comment:
i know this is just a draft for now, but when you're run please be sure not to commit _new_ exceptions into this file. Our goal is to remove betterer when these get down to zero, which we can't do if we add new exceptions.

prefer accurately typing new code, or using `// eslint-ignore` comments if there's absolutely no way to do that.